src/api/index.ts

```diff
@@ -1,22 +1,32 @@
-import { Network, TokenApi, DepositApi } from 'types'
+import { Network, TokenApi, WalletApi, DepositApi } from 'types'
+import { WalletApiMock } from './WalletApi/WalletApiMock'
 import { TokenApiImpl } from './TokenApi/TokenApiImpl'
 import { DepositApiMock } from './ExchangeApi/mock/DepositApiMock'
-import balanceStates from '../../test/data/balanceStates'
+import { balanceStates } from '../../test/data'
 
 const isMock = process.env.MOCK === 'true'
 
 function createTokenApi(): TokenApi {
   return new TokenApiImpl([Network.Mainnet, Network.Rinkeby])
 }
 
+function createWalletApi(): WalletApi {
+  if (isMock) {
+    return new WalletApiMock()
+  } else {
+    throw new Error('Only has been implemented')
```

Comment:
Only what? :)

Reply:
Yes, this was corrected after you mentioned it in a PR. It's solved in https://github.com/gnosis/dex-react/pull/57/commits/f672573b33711bdcffacfd0ee204c8f58c9ae451